content/docs/cli/environment-variables.md

```diff
@@ -111,6 +155,127 @@ aliases:
         </p>
         <pre><code class="text-xs">PULUMI_DEBUG_PROMISE_LEAKS=true</code></pre>
     </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DEV
+        </span>
+    </dt>
+    <dd>
+        <p>
+            Enable features for hacking on pulumi itself.
+        </p>
+        <pre><code class="text-xs">PULUMI_DEV=true</code></pre>
+    </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION
+        </span>
+    </dt>
+    <dd>
+        <p>
+            Disables the automatic installation of missing plugins.
+        </p>
+        <pre><code class="text-xs">PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION=true</code></pre>
+    </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DISABLE_PROVIDER_PREVIEW
+        </span>
+    </dt>
+    <dd>
+        <p>
+            Disables provider preview and enabled previous more concervative preview behavior.
+        </p>
+        <pre><code class="text-xs">PULUMI_DISABLE_PROVIDER_PREVIEW=true</code></pre>
+    </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DISABLE_RESOURCE_REFERENCES
+        </span>
+    </dt>
+    <dd>
+        <p>
+            Disables resource references in Invoke gRPC calls.
+        </p>
+        <pre><code class="text-xs">PULUMI_DISABLE_RESOURCE_REFERENCES=true</code></pre>
+    </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DISABLE_OUTPUT_VALUES
```

Comment:
Similar to `PULUMI_DISABLE_RESOURCE_REFERENCES`, though I'm not sure we're quite ready to get rid of it. I don't think we're going to want to maintain this long term, so not sure we should doc it here.